dec/decode.c

```diff
@@ -762,7 +762,7 @@ static void DecodeBlockTypeWithContext(BrotliState* s,
   s->context_lookup2 = &kContextLookup[kContextLookupOffsets[context_mode + 1]];
 }
 
-BrotliResult WriteRingBuffer(BrotliOutput output,
+static BrotliResult WriteRingBuffer(BrotliOutput output,
```

Comment:
Please fix the indentation of the arguments (here and below).

You can add static to  BrotliAllocateRingBuffer as well.